packages/header-parser/src/index.ts

```diff
@@ -230,7 +230,7 @@ const typeScriptVersionLineParser: pm.Parser<AllTypeScriptVersion> = pm
 const typeScriptVersionParser: pm.Parser<AllTypeScriptVersion> = pm
   .regexp(/\r?\n/)
   .then(typeScriptVersionLineParser)
-  .fallback<TypeScriptVersion>("3.1");
+  .fallback<TypeScriptVersion>("3.2");
```

Comment:
Any reason not to do `TypeScriptVersion.shipped[0]`?

Reply:
Good idea!